src/system/system_win.ml

```diff
@@ -38,10 +38,13 @@ let fspathAddSuffixToFinalName f suffix = f ^ suffix
 (****)
 
 let fixPath f =
-  for i = 0 to String.length f - 1 do
-    if f.[i] = '/' then Bytes.set f i '\\'
-  done;
-  f
+  let length = String.length f in
+  let buffer = Bytes.create length in
+    String.blit f 0 buffer 0 length;
+    for i = 0 to length - 1 do
+      if Bytes.get buffer i = '/' then Bytes.set buffer i '\\'
+    done;
+    Bytes.to_string buffer
```

Comment:
The indenting is misleading here; please un-indent this block to make it align with its surrounding `let ... in`.

Reply:
Just the last line, correct?

Reply:
Done.